packages/renderer/src/lib/pod/DeployPodToKube.svelte

```diff
@@ -569,7 +569,7 @@ function updateKubeResult() {
               <ul class="list-disc list-inside">
                 {#each createdRoutes as createdRoute}
                   <li class="pt-2">
-                    Port {createdRoute.spec.port.targetPort} is reachable with route
+                    Port {createdRoute.spec.port?.targetPort} is reachable with route
```

Comment:
I think you will need to wrap this text with a `{#if createdRoute.spec.port}`, or the text will be meaningless

Reply:
we are the one that creates the route when deploying a pod so i expect that a port is always there as before. This is just a fix bc lint was complaining.

I added support about the path because when listing all routes from a cluster  there could be some resources not created by us which just uses path.